client/src/screens/training/style.module.scss

```diff
@@ -0,0 +1,65 @@
+@use "../../components/common/_colors";
+
+.screen {
+  height: 85%;
+  display: flex;
+  overflow: auto;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 10px;
+}
+
+.buttonScreen {
+  height: 12%;
+  display: flex;
+  justify-content: center;
+}
+
+.navbarContainer.navbarContainer {
+  justify-content: space-between;
+}
+
+.topBar {
+  padding: 20px;
+  display: flex;
+  justify-content: center;
+}
+
+.button {
+  flex: 0 0 auto;
+  margin: auto;  
+  width: 150px;
+}
+
+.exercises {
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 20px 0 20px;
+  margin: 0;
+}
+
+.exercise {
+  display: flex;
+  align-items: center;
+
+  & + & {
+    margin-top: 20px;
+  }
+}
+
+.exerciseBox {
+  flex: 1 1 auto;
+  padding: 10px;
+  border: 1.5px dashed colors.$regular;
+  border-radius: 4px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.sets {
+  width: 80%;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
```

Comment:
did you just copy-pasted css? or did some clean up for you needs?

Reply:
did some cleaning up for my needs